src/HistoryDialog.vala

```diff
@@ -114,6 +118,11 @@ namespace PantheonCalculator {
         }
 
         private void on_response (Gtk.Dialog source, int response_id) {
+            if (response_id == Gtk.ResponseType.ACCEPT) {
```

Comment:
It feels a little odd having both "ACCEPT" and "OK" responses although it works.  Maybe consider defining our response id for "Clear History"?  Any positive integer will do.

Reply:
I think you're right, I've just updated the PR.